src/map/mob.cpp

```diff
@@ -4322,7 +4322,7 @@ uint64 MobDatabase::parseBodyNode(const YAML::Node &node) {
 		if (!this->asString(node, "AegisName", name))
 			return 0;
 
-		if (name.length() > NAME_LENGTH) {
+		if (name.size() > NAME_LENGTH) {
```

Comment:
"Both string::size and string::length are synonyms and return the same value."
https://www.cplusplus.com/reference/string/string/size/

Is there a reason why we change it? Or just to always use size?

Reply:
Yeah I think I used length at the time as I was typing and thinking differently. Since we use size elsewhere I was just trying to keep it harmonized!